README.rst

```diff
@@ -82,70 +82,72 @@ PyPI wheels (pip)
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ pip install pennylane-lightning[gpu]
-    $ pip install pennylane-lightning[kokkos]
+    pip install pennylane-lightning[gpu]
+    pip install pennylane-lightning[kokkos]
 
 
 Install from source
 ===================
 
 To build Lightning plugins from source you can run
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
+    PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
 
-where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu`` or ``lightning_kokkos``.
+where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu``,  ``lightning_kokkos``, or ``lightning_tensor``.
 The `pybind11 <https://pybind11.readthedocs.io/en/stable/>`_ library is required to bind the C++ functionality to Python.
 
 A C++ compiler such as ``g++``, ``clang++``, or ``MSVC`` is required.
 On Debian-based systems, this can be installed via ``apt``:
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ sudo apt -y update &&
-    $ sudo apt install g++ libomp-dev
+    sudo apt -y update && sudo apt install -y g++ libomp-dev
 
 where ``libomp-dev`` is included to also install OpenMP.
 On MacOS, we recommend using the latest version of ``clang++`` and ``libomp``:
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ brew install llvm libomp
+    brew install llvm libomp
 
-The Lightning-GPU backend has several dependencies (e.g. ``CUDA``, ``custatevec-cu12``, etc.), and hence we recommend referring to Lightning-GPU installation section.
-Similarly, for Lightning-Kokkos it is recommended to configure and install Kokkos independently as prescribed in the Lightning-Kokkos installation section.
+The Lightning-GPU backend has several dependencies (e.g. ``CUDA``, ``custatevec-cu12``, etc.), and hence we recommend referring to `Lightning-GPU installation <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_gpu/installation.html>`_ section.
+Similarly, for Lightning-Kokkos it is recommended to configure and install Kokkos independently as prescribed in the `Lightning-Kokkos installation <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_kokkos/installation.html>`_ section.
 
 Development installation
 ========================
 
 For development and testing, you can install by cloning the repository:
 
-.. code-block:: console
+.. code-block:: bash
 
-    $ git clone https://github.com/PennyLaneAI/pennylane-lightning.git
-    $ cd pennylane-lightning
-    $ pip install -r requirements.txt
-    $ PL_BACKEND=${PL_BACKEND} python scripts/configure_pyproject_toml.py
-    $ pip install -e . -vv
+    git clone https://github.com/PennyLaneAI/pennylane-lightning.git
+    cd pennylane-lightning
+    pip install -r requirements.txt
+    PL_BACKEND=${PL_BACKEND} python scripts/configure_pyproject_toml.py
+    pip install -e . -vv
```

Comment:
```suggestion
    pip install -e . --config-settings editable_mode=compat
```
and could you please change it accordingly everywhere in the documentation?
It would be great if you could also update it in the Makefile!